### ISSUE TITLE
[NCryptDecrypt - 0x80280095] TPM_20_E_SIZE After plugin upgrade

### ISSUE THREAD

**Issue by shuffle-c** [sirAndros/KeePassWinHello#77]
```
KeePassWinHello.AuthProviderSystemErrorException: NCryptDecrypt
Error code: 0x80280095
   at KeePassWinHello.WinHelloProvider.SECURITY_STATUS.ThrowOnError(String name, Int32 ignoreStatus)
   at KeePassWinHello.WinHelloProvider.PromptToDecrypt(Byte[] data, Boolean retry)
   at KeePassWinHello.WinHelloProvider.PromptToDecrypt(Byte[] data)
   at KeePassWinHello.KeyCipher.UnProtect(ProtectedBinary key)
   at KeePassWinHello.ProtectedKey.GetCompositeKey(KeyCipher keyCipher)
   at KeePassWinHello.KeyManager.ExtractCompositeKey(String dbPath, CompositeKey& compositeKey)
   at KeePassWinHello.KeyManager.Unlock(KeyPromptForm keyPromptForm, String dbPath)
   at KeePassWinHello.KeyManager.OnKeyPrompt(KeyPromptForm keyPromptForm, MainForm mainWindow)
   at KeePassWinHello.KeePassWinHelloExt.OnWindowAdded(Object sender, GwmWindowEventArgs e)
```

## Context

Please complete the following information:

| Component | Version |
|-----------|---------|
| Operating System | Windows 10 19043.1165 |

**Hardware info**

Hardware TPM
![MicrosoftTeams-image (5)](https://user-images.githubusercontent.com/3392944/130130544-4be49946-4eac-4ffb-8c21-7531b8e3643c.png)
 
 **Plugin settings**
 
 - [x] Is persistent mode enabled ("Store keys in Windows Credential Manager" checked)
 
 **KeePass settings and environment**
 
 - [x] Is secure desktop enabled
 - [ ] Is KeePass running under Administrator (in elevated process)
 - [ ] Was it right after hibernation
 - [x] Was KeePass minified or closed
 
_On behalf of @sirAndros_

**Comment by sirAndros** [comment]
Fixed in [v3.2](https://github.com/sirAndros/KeePassWinHello/releases/tag/v3.2)

**Comment by sirAndros** [comment]
Actually, it will reconfirm not after plugin upgrade but after change debug binaries change to release one
